src/MusicStore/Program.cs

```diff
@@ -66,6 +70,39 @@ public static void Main(string[] args)
                 requestTime.Stop();
                 Console.WriteLine("Response: {0}", response.StatusCode);
                 Console.WriteLine("Request took {0}ms", requestTime.ElapsedMilliseconds);
+                
+                var minRequestTime = long.MaxValue;
```

Comment:
Can you update this to print the total time for cold start (server started + first request) and then a few blank lines before popping off to do requests? I want to make sure that the measurement of cold start doesn't get lost in all of the new output, and it's always been confusing that you're supposed to sum (server started + first request0